public/mojolicious.css

```diff
@@ -113,3 +113,8 @@ ul { list-style-type: square }
   height: 450px;
   text-align: center;
 }
+@media only screen and (max-width: 798px) {
+    #mojo-fun a img {
+        width: 100%;
+    }
+}
```

Comment:
Inconsistent indentation.

Reply:
I think it would be better to have a global rule like this:

```
img {
  max-width: 100%;
}
```

Reply:
At some point this happens with a global rule, no?
![3 width FF iPadPro_L](https://user-images.githubusercontent.com/313562/95727736-1719c980-0cd7-11eb-87a7-c4c055c3a13f.png)

Reply:
That's unexpected... I guess your change is good then.